src/app/app.component.ts

```diff
@@ -123,7 +123,7 @@ export class AppComponent implements OnInit, OnDestroy {
     podcast.doc.followItems('prx:stories', {
       per: podcast.doc.count('prx:stories'),
       filters: 'v4',
-      sorts: 'released_at: desc, published_at: desc'
```

Comment:
there's no reason to do this episode sort on the request since it's done in the reducer, one less burden for CMS